model_zoo/vlm/api_model/model_adapter.py

```diff
@@ -70,9 +70,36 @@ def model_init(self, task_info: Dict):
             "max_num_frames",
             "stream",
             "system_prompt",
+            "num_infers",
         ]
         model_config = {k: task_info[k] for k in model_config_keys if k in task_info}
 
+        # Parse extra_args and add any parameters that exist in model_config_keys
+        if task_info.get("extra_args"):
```

Comment:
Verify that modifying extra_args does not have side effects in the `ModelServer`